modules/ui/src/main/resources/lucuma-css/lucuma-ui.scss

```diff
@@ -102,7 +98,7 @@ button.p-component.p-button {
   &.p-button-sm,
   &.p-button-lg {
     &:not(.pl-compact) {
-      padding: var(--pl-padding);
+      padding: var(--pl-button-padding);
```

Comment:
Search/replace error from a different PR. Added the `pl` prefix because it seemed like the thing to do.